src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Constants.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal partial class SymbolDisplayVisitor
     {
-        private void AddConstantValue(ITypeSymbol type, object constantValue, bool preferNumericValueOrExpandedFlagsForEnum = false)
+        private void AddConstantValue(ITypeSymbol type, object constantValue, SymbolDisplayFormat format, bool preferNumericValueOrExpandedFlagsForEnum = false)
```

Comment:
>SymbolDisplayFormat format [](start = 78, length = 26)

Are there callers that pass a value other than the `format` field?

Reply:
I didn't realize `format` was a field and not a local (it doesn't follow naming conventions). I've now amended the previous commit to use it directly instead of pass it as an argument.